We want the tests to exit with an error code if they fail

--- tests/runtests.orig	2022-06-09 09:23:35.000000000 -0400
+++ tests/runtests	2022-06-09 09:26:45.000000000 -0400
@@ -71,6 +71,7 @@
   echo >&2 "                  / of which $hmakeok known / $hmakebad bad"
   echo >&2 "$outok outputs match / $outbad outputs differ"
   echo >&2 "$errok errors  match / $errbad errors  differ"
+  if [[ $outbad -gt 0 || $errbad -gt 0 || $hmakebad -gt 0 ]]; then exit 1; fi
 }
 
 clean () {
